sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live`
```

Comment:
This view replaces the existing `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live` view. The advantage of simply replacing it is that queries that query the view won't break (like the queries currently implemented in Grafana)